README.md

```diff
@@ -38,6 +38,19 @@ See [index.js](https://github.com/mifi/instauto/blob/master/index.js) for availa
 
 See [example.js](https://github.com/mifi/instauto/blob/master/example.js) for example of features
 
+## Docker compose by https://github.com/maxxxdj
+
+- We've added a Docker support for insta-auto!
```

Comment:
The name of the project is `instauto`, so I'd prefer to keep that name instead of `insta-auto`